lib/ui/LandingPage.dart

```diff
@@ -124,117 +127,99 @@ class LandingPage extends StatelessWidget {
   final LandingPageViewModel _viewModel;
   final LandingPageStyle _style;
 
-  const LandingPage({Key key,
-    LandingPageViewModel viewModel,
-    LandingPageStyle style = _defaultStyle})
+  const LandingPage(
+      {Key key,
+      LandingPageViewModel viewModel,
+      LandingPageStyle style = _defaultStyle})
       : this._viewModel = viewModel,
         this._style = style,
         super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      body: SingleChildScrollView(
-        child: Column(
-          children: <Widget>[
-            _buildHeader(),
-            _buildSubtitle(),
-            SizedBox(
-              height: _style.subtitleLineSpace
-            ),
-            _buildDetailText(),
-            SizedBox(
-              height: _style.detailTextLineSpace
-            ),
-            _buildAction(context),
-            SizedBox(
-              height: _style.actionLineSpace
-            ),
-            _buildFooter(context),
-          ],
-        ),
+    return SingleChildScrollView(
```

Comment:
Move this scrollview as a child of the `PlaygroundWidget` and add this widget as a child of the `SingleChildScrollView`